vip-plugins/vip-plugins.php

```diff
@@ -169,13 +169,13 @@ function wpcom_vip_plugin_action_links( $actions, $plugin_file, $plugin_data, $c
 function wpcom_vip_option_active_plugins( $value, $option ) {
 	$code_plugins = wpcom_vip_get_filtered_loaded_plugins();
 
-	if ( ! empty( $value ) && ! empty( $code_plugins ) ) {
-		$value = array_merge( $value, $code_plugins );
+	if ( false === is_array( $value ) ) {
+		$value = array();
 	}
 
-	if ( ! empty( $value ) ) {
-		sort( $value );
-	}
+	$value = array_unique( array_merge( $value, $code_plugins ) );
```

Comment:
I know that I have already approved the PR, I just wonder why this `array_unique` is necessary here, while it is not needed in the `wpcom_vip_site_option_active_sitewide_plugins ` ?

Reply:
Once this discrepancy gets resolved, we also might want to merge this function with the `wpcom_vip_site_option_active_sitewide_plugins` and have a single one instead of two identical functions.

Reply:
The array unique removes duplicates (plugins activated by both code and the UI). It is not needed on sitewide plugins as the array structure is different (keyed by a timestamp). It is possible that two timestamps could be the same and therefore we would not able to make that array  unique.